CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## [project-title] Changelog
+## Azure Search .NET Samples Changelog

```

Comment:
We don't typically use changelog files. Git history is a much better way to track changes in a repo. I think you can just delete this file in all the new sample repos.